test/e2e/anti_affinity_test.go

```diff
@@ -0,0 +1,188 @@
+/*
```

Comment:
rename file to `clustermodules_test.go`

Reply:
This test is verifying the anti affinity setup for CAPV created VMs, hence I feel the name is appropriate. That is the general model we have followed for other tests.